caffe2/operators/accuracy_op.cc

```diff
@@ -14,21 +14,43 @@ bool AccuracyOp<float, CPUContext>::RunOnDevice() {
   DCHECK_EQ(label.dim32(0), N);
   Y->Resize(vector<TIndex>());
   const auto* Xdata = X.data<float>();
-  const auto* labeldata = label.data<int>();
+  const auto* labelData = label.data<int>();
+  const int top_k = top_k_;
```

Comment:
Or better, do CAFFE_ENFORCE(top_k <= D, "Hey dude"); to guard against incorrect arguments.